safe-global/web-core
# Fix: sandbox safe app icons

## What it solves

Loading Safe App icons directly inadvertently tracks the user on those websites, which is a security/privacy concern.

This PR wraps icons in sandboxed iframes and this effectively prevents 3rd-party cookies from being set.

I've tested this in Safari, Chrome, and Firefox.

## How to test
On dev or prod:
* Open the app, clear the cookies, local storage, etc
* Open the Dashboard and/or the Apps page
* Observe that a lot of 3rd-party cookies (e.g. Zerion) are set on their corresponding domains.

On this branch:
* External cookies will not be set unless you open a Safe App

See [this doc](https://www.notion.so/gnosis-safe/Third-party-cookies-tracking-through-Safe-App-Icons-fd8cc28e3ccb4de7a515eeee25c3b5cb) for details.
